### PR TITLE
[fix](docker) add metaservice start console to docker entrypoint

### PR DESCRIPTION
**Pull request by catpineapple**
### What problem does this PR solve?

1. add metaservice start console to docker entrypoint. we can see the log details through `kubectl logs`
2. add dcr workload group support to be_entrypoint. 

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Check List (For Committer)

- Test 

    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [x] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No colde files have been changed.
        - [x] Other reason 

- Behavior changed:

    - [x] No.
    - [ ] Yes. 

- Does this need documentation?

    - [x] No.
    - [ ] Yes. 

- Release note

    
    
    None

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by catpineapple** [comment]
run buildall

**Review by intelligentfu8 — DISMISSED** [review]

**Comment by catpineapple** [comment]
run buildall

File: docker/runtime/be/resource/be_entrypoint.sh [hunk 1]
```diff
@@ -65,6 +76,11 @@ update_conf_from_configmap()
             # make a backup
             mv -f $tgt ${tgt}.bak
         fi
+        if [[ "$conffile" == "be.conf" ]]; then
+             cp $CONFIGMAP_MOUNT_PATH/$conffile $DORIS_HOME/conf/$file
+             add_cluster_info_to_conf
+             continue
+        fi
         ln -sfT $CONFIGMAP_MOUNT_PATH/$conffile $tgt
     done
```
Comment (intelligentfu8) [inline docker/runtime/be/resource/be_entrypoint.sh:85]: move add_cluster_info_to_conf to here, only one code.

**Comment by catpineapple** [comment]
run buildall

**Review by intelligentfu8 — APPROVED** [review]
LGTM

**Review by zy-kkk — APPROVED** [review]

**Comment by shuke987** [comment]
skip buildall

**Pull request merged**